cpukit/score/src/thread.c

```diff
@@ -29,8 +29,9 @@
 
 THREAD_OFFSET_ASSERT( Object );
 THREAD_OFFSET_ASSERT( current_state );
-THREAD_OFFSET_ASSERT( current_priority );
-THREAD_OFFSET_ASSERT( real_priority );
+/* THREAD_OFFSET_ASSERT( Priority_node.current_priority );
+THREAD_OFFSET_ASSERT( Priority_node.real_priority );
+*/
```

Comment:
Probably, this needs to be `THREAD_OFFSET_ASSERT( Priority_node );` because we replace the two fields of the struct by the one embedded struct.